plugins/chat/assets/javascripts/discourse/components/chat-drawer/channel-threads.gjs

```diff
@@ -10,7 +10,7 @@ import ChatDrawerHeaderRightActions from "discourse/plugins/chat/discourse/compo
 import ChatDrawerHeaderTitle from "discourse/plugins/chat/discourse/components/chat-drawer/header/title";
 import ChatThreadList from "discourse/plugins/chat/discourse/components/chat-thread-list";
 
-export default class ChatDrawerThreads extends Component {
+export default class ChatDrawerChannelThreads extends Component {
```

Comment:
SO the general idea I will be following now in chat is:

- components
  - chat <- the base templates for chat routes
    - chat-threads.gjs
    - chat-channel.gjs
    - ...
  - chat-drawer <- the base templates for drawer routes
    - chat-threads.gjs
    - chat-channel.gjs
  - component-a
  - component-b
  - component-c
  - ...